.rubocop.yml

```diff
@@ -0,0 +1,16 @@
+Style/Documentation:
+  Enabled: false
+
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles.
+Style/StringLiterals:
+  Enabled: false
+
+Metrics/LineLength:
+  Max: 80
```

Comment:
This is incomplete, there are two more directives that control line length warnings. https://github.com/spartansystems/promox_spree/blob/master/.rubocop.yml

You're also overriding a default value with the default value. Why?